### PR TITLE
#19 Shift Logger

### LINKED ISSUE #19: Shift Logger
This issue addresses the content of the Shift Logger page. This page should explain the logger widget and how it can be configured through environment variables. This includes:
- Overview of the logger widget
- Log levels available and colors
- How to set the debug level in the UI
- Where to find Shift logs
- How to configure the logger through environment variables
  - Default log level
  - Log location
  - Log file size
  - Log file number

Screenshots should be added where necessary.

### PR DESCRIPTION
**Pull request by juliavisserinbibo**
## Issue
Closes #19 

## Description 
This PR addresses the content of the Shift Logger page. This page provides a more detailed explanation of the logger and how it can be configured through environment variables. This includes:

- Log levels available and colors
- How to set the debug level in the UI
- Where to find Shift logs
- How to configure the logger through environment variables

### CODE REVIEW

File: docs/reference/logger.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+
+All messages caught by the Logger are recorded in log files. Shift uses a rotating file handler, meaning that new logs will be created when the file exceeds a specific size without exceeding a set number of backups. By default, the log files can be found in a temporary folder in the user's AppData directory.
+
+## Configuring the Logger
+
+The default configuration settings of the Logger can be changed by defining the following environment variables:
+
+|Environment Variable| Description | Fallback Value |
+|:--|:---|:--|
+|`SHIFT_LOG_LEVEL`| The default log level when starting Shift| "*WARNING*"
```
Comment (hectorjimenezinbibo) [inline docs/reference/logger.md:36]:
```suggestion
|`SHIFT_LOG_LEVEL`| The default log level when starting Shift.| "*WARNING*"

```

**Review by hectorjimenezinbibo — CHANGES_REQUESTED** [review]

**Review by hectorjimenezinbibo — APPROVED** [review]

File: docs/reference/logger.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+The default configuration settings of the Logger can be changed by defining the following environment variables:
+
+|Environment Variable| Description | Fallback Value |
+|:--|:---|:--|
+|`SHIFT_LOG_LEVEL`| The default log level when starting Shift.| "*WARNING*"
+
+|`SHIFT_LOG_MAXSIZE`| The maximum size of the log files in MB.| "*10485760*"
+|`SHIFT_LOG_MAXFILES`| The maximum number of log files.| "*5*"
+|`SHIFT_LOG_DIRECTORY`| The location of log files.| "*C:/Users/[USERNAME]/AppData/Local/Temp/Shift*"
```
Comment (albertochimentiinbibo) [inline docs/reference/logger.md:40]:
```suggestion
|`SHIFT_LOG_DIRECTORY`| The location of log files.| "*C:/Users/[USERNAME]/AppData/Local/Temp/Shift*" (only in Windows)

```

**Review by albertochimentiinbibo — CHANGES_REQUESTED** [review]

**Review by JorgeHernandezibanezinbibo — APPROVED** [review]

File: docs/reference/logger.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+By default, when opening Shift the Logger is set to the Warning level. The level can be changed through the UI, by choosing the preferred log level under the `View -> Log Level` options in the menu top bar.
+
+>[!NOTE]
+>If persistent critical errors are experienced, please, [contact support](https://inbibo.co.uk/contact?_gl=1*m7h22j*_up*MQ..*_ga*MTU4MzcxNTg4OS4xNzI0MzE2MTQ2*_ga_DY05R1LZDB*MTcyNDMxNjE0Ni4xLjEuMTcyNDMxNjE1Mi4wLjAuMTg0MTg0MzEwNw..) attaching the error message and providing a description of the issue.
+
+
+## Log Files
+
+All messages caught by the Logger are recorded in log files. Shift uses a rotating file handler, meaning that new logs will be created when the file exceeds a specific size without exceeding a set number of backups. By default, the log files can be found in a temporary folder in the user's AppData directory.
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/reference/logger.md:28]: I forgot this is specific to windows... we should add Linux too?

[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/reference/logger.md]: I added a box with the example
image

File: docs/reference/logger.md (hunk 2 above)
Comment (juliavisserinbibo) [reply docs/reference/logger.md]: I changed to a description

**Review by leegeertseninbibo — APPROVED** [review]

**Review by albertochimentiinbibo — APPROVED** [review]

**Pull request merged**